weaveworks/weave-gitops-enterprise
# Fixes helm-charts appearing on the add-app page

- We only conditionally load the charts if the source is a helm-release
- We weren't setting the source type properly so this completely fails
- Fix this here so the charts are again loaded

Intro'd in #2069 


Closes #2280 


**What changed?**

Correctly set the source_type when selecting a gitrepo/helmrepo, so that we can check whether we should load the profiles or not.


**Why was this change made?**

Fix the bug!


**How was this change implemented?**



**How did you validate the change?**

manually

To repro:
- https://demo-01.wge.dev.weave.works/applications/create?clusterName=management
- Select and helm repo and see an empty table :(
- `PROXY_HOST=https://demo-01.wge.dev.weave.works/ yarn start`
- http://localhost:3000/applications/create?clusterName=management
- select a helm repo and see the charts appear :)


**Release notes**



**Documentation Changes**
